src/Tribe/Importer/Column_Mapper.php

```diff
@@ -37,10 +37,17 @@ public function set_defaults( $defaults ) {
 		$this->defaults = $defaults;
 	}
 
+	/**
+	 * Render a select dropdown at the top of a column for column mapping.
+	 *
+	 * @param int $index The index of the column. Starts with 0.
+	 *
+	 * @return string The HTML markup of the select box.
+	 */
 	public function make_select_box( $index ) {
 		$selected = isset( $this->defaults[ $index ] ) ? $this->defaults[ $index ] : '';
 		$html     = '<select name="column_map[' . $index . ']">';
-		$html .= '<option value="">' . esc_html__( 'Do Not Import', 'the-events-calendar' ) . '</option>';
+		$html     .= '<option value="">' . esc_html__( 'Do Not Import', 'the-events-calendar' ) . '</option>';
```

Comment:
it's weird, but the `=` must align so it would be:
```suggestion
		$html    .= '<option value="">' . esc_html__( 'Do Not Import', 'the-events-calendar' ) . '</option>';
```

Reply:
Yeah, was wondering... Thx, committed.